src/quo2/components/numbered_keyboard/numbered_keyboard/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.numbered-keyboard.numbered-keyboard.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.numbered-keyboard.keyboard-key.view :as keyboard-key]
+            [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.style :as style]))
+
+(defn keyboard-item
+  [{:keys [item type disabled? on-press blur? theme]
+    :or   {item nil}}]
+  [keyboard-key/view
+   {:disabled? disabled?
+    :on-press  on-press
+    :blur?     blur?
+    :theme     theme
+    :type      type} item])
+
+(defn- view-internal
+  []
+  (fn [{:keys [disabled? theme blur? left-action delete-key? on-press]}]
+    [rn/view
+     {:style style/container}
+     (for [row-index (range 1 4)]
+       ^{:key row-index}
+       [rn/view {:style style/row-container}
+        (for [column-index (range 1 4)]
+
+          [keyboard-item
+           {:item      (+ (* (dec row-index) 3) column-index)
+            :type      :digit
+            :disabled? disabled?
+            :on-press  on-press
+            :blur?     blur?
+            :theme     theme}])])
+     ;; bottom row
+     [rn/view {:style style/row-container}
+      (condp = left-action
+        :dot     [keyboard-item
+                  {:item      "."
+                   :type      :digit
+                   :disabled? disabled?
+                   :on-press  on-press
+                   :blur?     blur?
+                   :theme     theme}]
+        :face-id [keyboard-item
+                  {:item      :i/face-id
+                   :type      :key
+                   :disabled? disabled?
+                   :on-press  on-press
+                   :blur?     blur?
+                   :theme     theme}]
+        :none    [keyboard-item])
```

Comment:
We could use `case` since you are using keywords to compare `left-action`, also we could avoid that `:none` and use the default case of `case`.

IMO it'd be a little simpler, since `case` is less complex to understand than `condp`, but the code looks pretty good, so feel free to apply or not this suggestion.

Reply:
Done